src/test/regress/sql/multi_schema_support.sql

```diff
@@ -69,3 +70,301 @@ SELECT master_append_table_to_shard(1190001, 'nation_local', 'localhost', :maste
 
 -- verify table actually appended to shard
 SELECT COUNT(*) FROM "nation._'append";
+
+
+-- test shard creation when search_path is set
+SET search_path TO test_schema_support;
+
+-- create shard with COPY on append distributed table
+CREATE TABLE nation_append2(
```

Comment:
Is this table created for just search path check? I couldn't see other references. Maybe we can rename it to nation_append_search_path, or something else...